frontend/packages/dev-console/src/components/helm/HelmReleaseResourceTableHeader.tsx

```diff
@@ -16,10 +16,10 @@ const HelmReleaseResourceTableHeader = () => {
       props: { className: tableColumnClasses.name },
     },
     {
-      title: 'Kind',
+      title: 'Type',
       sortField: 'kind',
       transforms: [sortable],
-      props: { className: tableColumnClasses.kind },
```

Comment:
switch to something like tableColumnClasses[0]

Reply:
you might be correct but somehow I feel `tableColumnClasses.kind` is easier to read compared to `tableColumnClasses[0]`

Reply:
Using an object to represent column header classes makes it much more stable and readable.